components/fxa-client/ios/FxAClient/FirefoxAccount.swift

```diff
@@ -48,7 +49,25 @@ public protocol PersistCallback {
 fileprivate let queue = DispatchQueue(label: "com.mozilla.firefox-account")
 
 open class FirefoxAccount: RustHandle {
-    fileprivate static var persistCallback: PersistCallback?
+    fileprivate var persistCallback: PersistCallback?
```

Comment:
I wonder if it's worth changing the persist fn to be a `throws` fn. It probably wants to do IO under the hood, I imagine.

Reply:
Interesting consideration. The `throw` on the callback will bubble back to `tryPersistState()` and I don't know that function could do much in terms of handling the error. Generally speaking, `throw` would be avoided on a callback.